packages/react/src/side-nav/SideNavUnorderedList.tsx

```diff
@@ -2,14 +2,18 @@ import { ReactNode } from 'react';
 import { Box } from '../box';
 import { LinkListContext, useLinkListDepth } from './context';
 
-type SideNavGroupProps = { children: ReactNode };
+type SideNavUnorderedListProps = { children: ReactNode; isOpen: boolean };
 
-export function SideNavGroup({ children }: SideNavGroupProps) {
+export function SideNavUnorderedList({
```

Comment:
I think we can remove the Unordered part in all this, because it's not particularly useful and increases cognitive load when reading the component name

Reply:
I don't feel too strongly either way, so all good